### PR TITLE
About page

### PR DESCRIPTION
**Pull request by krystxf**
- stats
- app links

### CODE REVIEW

File: src/pages/about.tsx [hunk 1]
```diff
@@ -1,11 +1,35 @@
...
+import { CountUp } from 'use-count-up';
 
 import { Layout } from '../components/Layout';
 
-const About = (props: { clipCount: number; version: string }): JSX.Element => {
+const About = (props: {
+  clipCount: number;
+  userCount: number;
+  version: string;
+}): JSX.Element => {
```
Comment (bigint) [inline src/pages/about.tsx:14]: React.ReactNode

Comment (bigint) [reply src/pages/about.tsx:14]: Also its good to use Interface for this!

File: src/pages/about.tsx [hunk 2]
```diff
@@ -45,20 +74,63 @@ const About = (props: { clipCount: number; version: string }): JSX.Element => {
...
   );
 };
 
+const Fact = ({
+  title,
+  value,
+}: {
+  title: string;
+  value: number;
+}): JSX.Element => {
```
Comment (bigint) [inline src/pages/about.tsx:116]: React.ReactNode

File: src/pages/about.tsx [hunk 3]
```diff
@@ -45,20 +74,63 @@ const About = (props: { clipCount: number; version: string }): JSX.Element => {
...
+  return (
+    <div className="text-center w-full cursor-pointer">
+      <h3 className="font-semibold text-3xl font-sans">
+        <CountUp isCounting end={value} duration={1.4} />
```
Comment (bigint) [inline src/pages/about.tsx:120]: 😅  is this real?

File: src/pages/about.tsx [hunk 4]
```diff
@@ -45,20 +74,63 @@ const About = (props: { clipCount: number; version: string }): JSX.Element => {
...
+      </h3>
+      {title}
```
Comment (bigint) [inline src/pages/about.tsx:122]: Wrap it to span or div 🙇🏻

**Review by bigint — CHANGES_REQUESTED** [review]

**Review by filiptronicek — CHANGES_REQUESTED** [review]
I really like the updates, looks great. One thing that I am curious about is why have the cursor be a pointer on the facts? I don't see why this is needed, did you want to direct the user to some more info/link? 
https://github.com/interclip/interclip-next/blob/988792539e45700dccbe40a36efe3802b72a0959/src/pages/about.tsx#L118

**Comment by krystofex** [comment]
Well I don't know really, where it should point to, it's just mistake haha

**Comment by filiptronicek** [comment]
@krystofex could you please revisit this PR and take a look at @bigint's comments? Also, please, as per my comment remove the pointer cursor.

**Comment by filiptronicek** [comment]
A bit sus @krystofex 

image

**Comment by filiptronicek** [comment]
@krystofex are you still going to work on this or should I close this and implement it later?

**Comment by krystofex** [comment]
I can finish it

**Review by filiptronicek — APPROVED** [review]
Great, thank you!

**Pull request closed without merging**